### ISSUE TITLE
BottomNavigationBar: unselected tab should have tab name as a11y label

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#19900]
Open the Bottom navigation demo in the gallery. Use talkback to focus on an unselected tab. It should announce the name of that tab, but it doesn't.
